website/docs/guides/getting_started.html.markdown

```diff
@@ -0,0 +1,176 @@
+---
+layout: "newrelic"
+page_title: "Getting Started with the New Relic provider"
+sidebar_current: "docs-newrelic-provider-getting-started"
+description: |-
+  Getting started with the New Relic provider
+---
+
+# Getting Started with the New Relic Provider
+
+## Before You Begin
+
+* This guide assumes you already have a New Relic agent deployed. If you don't have New Relic integrated yet, check out [New Relic's introduction documentation](https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/introduction-new-relic) to get started there, then head back over here to get started with the New Relic Terraform Provider.
+* [Install Terraform](https://www.terraform.io/intro/getting-started/install.html)
+and read the Terraform getting started guide that follows. This guide will
+assume a basic understanding of Terraform.
```

Comment:
nit: unnecessary line breaks?

Reply:
Not sure which line break you're referring to here, but Terraform's markdown flavor has several subtle differences with how it parses the markdown - line breaks being a part of that.